docker/docker-compose-dev.yaml

```diff
@@ -0,0 +1,35 @@
+# docker-compose -f ./docker/docker-compose-dev.yaml up --build -d
+# docker-compose -f ./docker/docker-compose-dev.yaml down
+
+version: "2.4"
+x-net: &net
+  networks:
+      - main
+services:
+  db:
+    <<: *net
+    image: datajoint/mysql:5.7
+    environment:
+      - MYSQL_ROOT_PASSWORD=simple
+  workflow-calcium-imagin-dev:
```

Comment:
```suggestion
  calcium-imaging-dev-workflow:
```

Reply:
Changing the name to be consistent with the other workflows.  We will hopefully eventually merge the `dev` and `test` Docker and Compose files.